Dockerfile

```diff
@@ -41,7 +41,7 @@ RUN chmod 755 /home/reset.sh
 ENV PATH=/root/bin:/root/.local/bin/:$PATH
 
 # The mounted repo should contain a build folder with the following files
-# 1) K8s config file as config
+# 1) K8s config file as config and Ansible's k8s module requires the openshift python package (>= 0.6).
```

Comment:
Don't think this is the right place to change. It should be added in docs/source/prerequisites.md rather